search-app/src/cmr/search/services/acls/acl_results_handler_helper.clj

```diff
@@ -41,11 +41,11 @@
   (let [{{entry-title :entry-title
           provider-id :provider-id
           access-value :access-value
-          start-date :start-date
-          end-date :end-date} :_source} elastic-result
-        start-date (parse-elastic-datetime start-date)
-        end-date (parse-elastic-datetime end-date)]
-
+          start-date1 :start-date
+          end-date1 :end-date} :_source} elastic-result
+        start-date (parse-elastic-datetime start-date1)
```

Comment:
maybe a better naming would be `start-date` and `elastic-start-date`

Reply:
done, but double check